Update to ratpack 1.6.0-rc-1 error

When I update my ratpack project to Ratpack 1.6.0-rc-1, there ware some errors because of missing commons-lang3 dependency, but handlebars.java 4.0.6 need commons-lang3.

https://github.com/ratpack/ratpack/blob/5c59ac116ea67cf6f774ec3dadc80fd79c7ebd74/ratpack-handlebars/ratpack-handlebars.gradle#L25

-----------------------------------------------------------------------------------
com.google.inject.ProvisionException: Unable to provision, see the following errors:

1) Error in custom provider, java.lang.NoClassDefFoundError: org/apache/commons/lang3/Validate
  at ratpack.handlebars.HandlebarsModule.provideTemplateCache(HandlebarsModule.java:176) (via modules: com.google.inject.util.Modules$OverrideModule -> com.google.inject.util.Modules$OverrideModule -> com.google.inject.util.Modules$OverrideModule -> com.google.inject.util.Modules$OverrideModule -> com.google.inject.util.Modules$OverrideModule -> scorpio.handlebars.HandlebarsModule)
  at ratpack.handlebars.HandlebarsModule.provideTemplateCache(HandlebarsModule.java:176) (via modules: com.google.inject.util.Modules$OverrideModule -> com.google.inject.util.Modules$OverrideModule -> com.google.inject.util.Modules$OverrideModule -> com.google.inject.util.Modules$OverrideModule -> com.google.inject.util.Modules$OverrideModule -> scorpio.handlebars.HandlebarsModule)
  while locating com.github.jknack.handlebars.cache.TemplateCache
    for the 4th parameter of ratpack.handlebars.HandlebarsModule.provideHandlebars(HandlebarsModule.java:184)
  at ratpack.handlebars.HandlebarsModule.provideHandlebars(HandlebarsModule.java:184) (via modules: com.google.inject.util.Modules$OverrideModule -> com.google.inject.util.Modules$OverrideModule -> com.google.inject.util.Modules$OverrideModule -> com.google.inject.util.Modules$OverrideModule -> com.google.inject.util.Modules$OverrideModule -> scorpio.handlebars.HandlebarsModule)
  while locating com.github.jknack.handlebars.Handlebars
    for the 1st parameter of scorpio.handlebars.render.HandlebarsTemplateRenderer.(HandlebarsTemplateRenderer.groovy:18)
  at sun.reflect.GeneratedMethodAccessor6.invoke(Unknown Source) (via modules: com.google.inject.util.Modules$OverrideModule -> com.google.inject.util.Modules$OverrideModule -> com.google.inject.util.Modules$OverrideModule -> com.google.inject.util.Modules$OverrideModule -> com.google.inject.util.Modules$OverrideModule -> scorpio.handlebars.HandlebarsModule)
  while locating scorpio.handlebars.render.HandlebarsTemplateRenderer
Caused by: java.lang.NoClassDefFoundError: org/apache/commons/lang3/Validate
	at com.github.jknack.handlebars.cache.ConcurrentMapTemplateCache.(ConcurrentMapTemplateCache.java:62)
	at com.github.jknack.handlebars.cache.ConcurrentMapTemplateCache.(ConcurrentMapTemplateCache.java:69)
	at ratpack.handlebars.HandlebarsModule.provideTemplateCache(HandlebarsModule.java:177)
	at ratpack.handlebars.HandlebarsModule$$FastClassByGuice$$cfbb43aa.invoke()

----------------------------------------------------------------------

We should update to handlebars.java v4.1.2 ( #1416 ), handlebars.java v4.1.2 removed commons-lang3 , antlr4-runtime, rhino dependencies.

--- a/ratpack-handlebars/ratpack-handlebars.gradle
+++ b/ratpack-handlebars/ratpack-handlebars.gradle
@@ -20,9 +20,8 @@ description = "Support for Handlebars server side template rendering in Ratpack 
 
 dependencies {
   compile project(":ratpack-guice")
-  compile('com.github.jknack:handlebars:4.0.6') {
+  compile('com.github.jknack:handlebars:4.0.7') {
     exclude group: "org.slf4j", module: "slf4j-api"
-    exclude group: "org.apache.commons", module: "commons-lang3"
   }
   runtime commonDependencies.slf4j
 }

